sample/src/main/res/layout/activity_main.xml

```diff
@@ -22,6 +22,7 @@
         app:dayBackgroundColor="#05000000"
         app:todayBackgroundColor="#1848adff"
         app:headerColumnBackground="#ffffffff"
-        app:todayHeaderTextColor="@color/accent"/>
+        app:todayHeaderTextColor="@color/accent"
+        app:showHalfHours="false"/>
```

Comment:
Change the function signature on the interpretTime method to "public String interpretTime(int hour, int minutes)". You can then add the "minutes" to the string that you are returning from that function.